applications/GeoMechanicsApplication/python_scripts/element_test/set_triaxial_test.py

```diff
@@ -47,7 +57,18 @@ def lab_test(dll_path, index, umat_parameters, num_steps, end_time, maximum_stra
 
     output_files = [os.path.join(tmp_folder, 'gid_output', "triaxial_Stage_2.post.res")]
     runner = TriaxialTestRunner(output_files, tmp_folder)
-    reshaped_values_by_time, vertical_strain, volumetric_strain, von_mise_stress, mean_effective_stresses = runner.run()#output_files, tmp_folder)
+    reshaped_values_by_time, vertical_strain, volumetric_strain, von_mise_stress, mean_effective_stresses = runner.run()
```

Comment:
I think we're missing a letter here
```suggestion
    reshaped_values_by_time, vertical_strain, volumetric_strain, von_mises_stress, mean_effective_stresses = runner.run()
```